src/ol_infrastructure/applications/micromasters/__main__.py

```diff
@@ -0,0 +1,174 @@
+micromasters_bucket = s3.Bucket(
+    f"micromasters-{stack_info.env_suffix}",
+    bucket=micromasters_bucket_name,
+    versioning=s3.BucketVersioningArgs(
+        enabled=True,
+    ),
+    tags=aws_config.tags,
+    acl="private",
+    policy=json.dumps(
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:GetObject",
+                        "s3:ListAllMyBuckets",
+                        "s3:ListBucket",
+                        "s3:ListObjects",
+                        "s3:PutObject",
+                        "s3:DeleteObject",
+                    ],
+                    "Resource": [f"arn:aws:s3:::{micromasters_bucket_name}/*"],
+                }
+            ],
+        }
+    ),
+    cors_rules=[{"allowedMethods": ["GET", "HEAD"], "allowedOrigins": ["*"]}],
+)
+
+
+micromasters_iam_policy = iam.Policy(
+    f"micromasters-{stack_info.env_suffix}-policy",
+    description="AWS access controls for the MicroMasters application in the "
+    f"{stack_info.name} environment",
+    path=f"/ol-applications/micromasters/{stack_info.env_suffix}/",
+    name_prefix=f"micromasters-{stack_info.env_suffix}-application-policy-",
+    policy=lint_iam_policy(
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": "s3:ListAllMyBuckets",
+                    "Resource": "*",
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:ListBucket*",
+                        "s3:PutObject",
+                        "s3:PutObjectAcl",
+                        "s3:GetObject*",
+                        "s3:DeleteObject*",
+                    ],
+                    "Resource": [
+                        f"arn:aws:s3:::{micromasters_bucket_name}",
+                        f"arn:aws:s3:::{micromasters_bucket_name}/*",
+                    ],
+                },
+            ],
+        },
+        stringify=True,
+        parliament_config={
+            "PERMISSIONS_MANAGEMENT_ACTIONS": {
+                "ignore_locations": [{"actions": ["s3:putobjectacl"]}]
+            }
+        },
+    ),
+)
+
+micromasters_vault_backend_role = vault.aws.SecretBackendRole(
+    "micromasters-app",
+    name="micromasters",
+    backend="aws-mitx",
+    credential_type="iam_user",
+    policy_arns=[micromasters_iam_policy.arn],
+)
+
+# Create RDS instance
+micromasters_db_security_group = ec2.SecurityGroup(
+    f"micromasters-db-access-{stack_info.env_suffix}",
+    description=f"Access control for the MicroMasters App DB in {stack_info.name}",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
```

Comment:
This is good, but will necessitate some port changes in other systems once we migrate. Namely Airbyte and Redash, because this had been using port 15432 for no good reason.

Reply:
Are there any action items for me around the necessity for that change? Should I file an issue? Make another CR in some other repo?

Guessing not and that you're just signposting, so I'll resolve this for now.